Cargo.toml

```diff
@@ -37,7 +37,7 @@ categories = ["development-tools", "wasm", "web-programming"]
 keywords = ["leptos"]
 version = "0.2.24"
 edition = "2021"
-rust-version = "1.71"
+rust-version = "1.82"
```

Comment:
This is due to the use of `is_none_or` in codebase and was part of clippy warnings.